docs/reference/google.auth.compute_engine.credentials.rst

```diff
@@ -1,5 +1,5 @@
-google.auth.compute_engine.credentials module
-=============================================
+google.auth.compute\_engine.credentials module
+==============================================
```

Comment:
Likewise here.  Note the [current, correctly-rendered page](https://google-auth.readthedocs.io/en/latest/reference/google.auth.compute_engine.html).

Reply:
Removed.